k8s/arroyo/templates/configmap.yaml

```diff
@@ -83,4 +87,4 @@ data:
           - name: arroyo-user-config
             mountPath: /config
           {{- end }}
-        command: "/app/arroyo-bin --config-dir /config worker"
\ No newline at end of file
+        command: ["/app/arroyo-bin --config-dir /config worker"]
```

Comment:
This still isn't quite right. With this, trying to run this in helm gives the error 

```
Configuration is invalid!
  • invalid type: found sequence, expected a string for key "default.kubernetes-scheduler.worker.command" in /root/.config/arroyo/config.yaml YAML file
```
It should be restored to what it currently is in master.